docs/manual.md

```diff
@@ -871,20 +871,24 @@ For more information see:
 
 ## Templates
 
-Goss test files can leverage golang's [text/template](https://golang.org/pkg/text/template/) to allow for dynamic or conditional tests.
+Goss test files can leverage golang's [text/template](https://golang.org/pkg/text/template/) to allow for dynamic or conditional tests. 
 
 Available variables:
 * `{{.Env}}`  - Containing environment variables
 * `{{.Vars}}` - Containing the values defined in [--vars](#global-options) file
 
-Available functions beyond text/template [built-in functions](https://golang.org/pkg/text/template/#hdr-Functions):
-* `mkSlice "ARG1" "ARG2"` - Retuns a slice of all the arguments. See examples below for usage.
-* `getEnv "var" ["default"]` - A more forgiving env var lookup. If key is missing either "" or default (if provided) is returned.
-* `readFile "fileName"` - Reads file content into a string, trims whitespace. Useful when a file contains a token.
-  * **NOTE:** Goss will error out during during the parsing phase if the file does not exist, no tests will be executed.
-* `regexMatch "(some)?reg[eE]xp"` - Tests the piped input against the regular expression argument.
-* `toLower` - Changes piped input to lowercase
-* `toUpper` - Changes piped input to UPPERCASE
+Available functions **in load order**:
```

Comment:
Not sure if "in load order" is clear to new users.

I wonder if there's a better way to word.

Reply:
That is fair, I struggled with the wording here. I tweak it by removing the `in load order` bit and explained at right after the list that some of Custom Goss Functions might overwrite the Sprig Functions and we are keeping the custom functions for backward compatibility.